platform/viewer/src/connectedComponents/ViewerRetrieveStudyData.js

```diff
@@ -26,19 +27,55 @@ class ViewerRetrieveStudyData extends Component {
 
   async loadStudies() {
     try {
-      const { server, studyInstanceUids, seriesInstanceUids } = this.props;
+      const { server, studyInstanceUids, seriesInstanceUIDs } = this.props;
+      const filters = {};
+
+      // Use the first, discard others
+      const seriesInstanceUID = seriesInstanceUIDs && seriesInstanceUIDs[0];
```

Comment:
Where `seriesInstanceUIDs` is set by the route?

Reply:
or from ViewerRouting or from useLocation hooks. I`m using the first strategy because it was already present on the code.